grove-beginner-kit/global.h

```diff
@@ -6,6 +6,7 @@
 #include "board.h"
 #if USE_NOTECARD
 #include <Notecard.h>
+extern Notecard notecard;
```

Comment:
Did you intend to go through the sensors and change them to from `NoteNewRequest()` to `notecard.newRequest()`?

Since you have put the mechanics in place, we probably should for the sake of consistency. Otherwise, if it's not worth it, then we could probably eliminate the global object.

Reply:
I only changed the minimum necessary to get it to compile, but good point on consistency, I can amend the other requests too.